### PR TITLE
Update year2018.md

### PR DESCRIPTION
**Pull request by nmohantyctr**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ x] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/releasenotes/archive/service/year2018.md [hunk 1]
```diff
@@ -6,3 +6,750 @@ description: This is an archive of 2018 Sumo Logic service release notes.
...
+
+New - Logs-to-Metrics is now GA! Transform your logs into metrics for high performance analytics using our Logs-to-Metrics solution. Gain 13-month retention and up to 100x query performance by parsing out or creating metrics from log streams in real-time.
+
+---
+## October 9, 2018 (SAML)
+
+Update - The Sumo page for configuring SAML for SSO (Administration > Security > SAML) has a new option: Disable Requested Authn Context. If you check this option, Sumo will not include the optional RequestedAuthnContext element of the SAML AuthnRequests it sends to your Idp. This is useful if your IdP does not support the RequestedAuthnContext SAML element. If you use Azure Active Directory for SSO, you should check the Disable Requested Authn Context option to prevent the following error message:
+
+SAML authentication request's RequestedAuthenticationContext's
+Comparison value must be "exact"
```
Comment (kimsauce) [inline docs/releasenotes/archive/service/year2018.md:171]:
```suggestion
SAML authentication request's RequestedAuthenticationContext's Comparison value must be "exact"
```

File: docs/releasenotes/archive/service/year2018.md [hunk 2]
```diff
@@ -6,3 +6,750 @@ description: This is an archive of 2018 Sumo Logic service release notes.
...
+
+The update provides a new dashboard, Log Spikes. In addition, new panels that present metric ingest in Data Points per Minute (DPM) were added to the Overview and Metrics dashboards.
+
+---
+## February 20, 2018 (Apps) 
+
+A new app was released today.
+
+Sumo Logic App for Kubernetes allows you to monitor Kubernetes deployments that are self-managed by open source tools such as Kops. The preconfigured dashboards present resource-related metrics at the Kubernetes pod, cluster, and namespace level; and provide operational insight into Kubernetes components, including nodes, the API Server, the Controller Manager, the Kube System, and the Scheduler. For more information, see Kubernetes.
+NoteDeprecated - The Sumo Logic App for Kubernetes was deprecated September 6, 2019, and replaced with the current Kubernetes App.
```
Comment (kimsauce) [inline docs/releasenotes/archive/service/year2018.md:662]: Add line break here and format the "Note" as an admonition

File: docs/releasenotes/archive/service/year2018.md [hunk 3]
```diff
@@ -6,3 +6,750 @@ description: This is an archive of 2018 Sumo Logic service release notes.
...
+ 
+The Sumo app for Google Cloud IAM is now available. The app gives you visibility into the activities in Cloud IAM. The preconfigured dashboards allow you to monitor the IAM project activities, operations, role activities, and policy changes.
+ 
+The Sumo app for Google Cloud VPC is now available. The app gives you visibility into activities, traffic, and VPC flows. The preconfigured dashboards provide details on VPC flows, source and destination IP addresses, ports, protocols, and messages.
+
+---
+## January 23, 2018 (Dashboard) 
+
+Bug Fix - Some dashboards were opening in Live Mode instead of Edit mode by default. This has been fixed and all dashboards open in Edit mode by default.
+Bug Fix - Some dashboards were not responding after selecting Open in a New Browser Tab. This has been fixed.
```
Comment (kimsauce) [inline docs/releasenotes/archive/service/year2018.md:706]: needs line break

**Review by kimsauce — CHANGES_REQUESTED** [review]
a few minor fixes

**Comment by kimsauce** [comment]
Hi @nmohantyctr  , per our convo this morning, I would close out this PR, since you're working on the correctly formatted version in Atom and pushing that as a PR.

**Comment by nmohantyctr** [comment]
Updating file locally.

**Pull request closed without merging**